Source/WebCore/accessibility/playstation/AXObjectCachePlayStation.cpp

```diff
@@ -88,7 +88,7 @@ void AXObjectCache::postPlatformNotification(AccessibilityObject& object, AXNoti
     RefPtr protectedObject = &object;
     switch (notification) {
     case AXNotification::AXSelectedChildrenChanged:
-        protectedObject = notifyChildrenSelectionChange(protectedObject.get());
+        protectedObject = dynamicDowncast<AccessibilityObject>(notifyChildrenSelectionChange(protectedObject.get()));
```

Comment:
Using `dynamicDowncast` means this can be null, and if it is, then we’ll dereference that null below when calling `postAccessibilityNotification`. We should probably add a null check. Or we could switch to use a `downcast` instead if it can’t ever be a non-`AccessibilityObject` (it will crash in that case which is not that different from the null dereference crash).

Reply:
I guess `downcast` should work; looks like the code previously just had `static_cast<AccessibilityObject*>`.